### ISSUE TITLE
Review powerplay properties available from 3rd party APIs after Powerplay 2.0 is released

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#2658]
## What happens now
Powerplay information has been revised to 2.0. 

## What I'd like to happen 
Powerplay fields and definitions available from 3rd party APIs may change.

## How it can happen
Review powerplay properties available from 3rd party APIs.
Update `StarSystem` properties as required... there is a new `ControllingPower` property in journal events in particular that we want to capture.

## EDDI Version
4.1.0-b3

**Comment by Darkcyde13** [comment]
Don't know if this is the right place to report this (apologies if it's not), but I've found that the `system.power` property seems to always be "None".  I've flown through a few enemy exploited systems after joining a power, and `system.power` has (so far) always been "None".  I only joined a short while ago, so I've not done much to test any power play stuff yet, to check anything else.

**Comment by Tkael** [comment]
Thanks. There are a few changes to the player journal with the latest update (though they are sparser than I would have wanted) and I'm working on getting a new beta revision out which takes those into account.
That said, powerplay properties only show up in the player journal at all if you are pledged to a power. If you haven't pledged then the player journal will have no information.

**Comment by Darkcyde13** [comment]
Yeah, I thought that would be the case, that's why I thought I'd report here, as I am pledged and the `system.power` was always 'None'.  I did check my `cmdr.power` and that correctly reported who I had aligned with, so I've assumed that any not working is probably due to the changes. :-)
